.github/workflows/ci.yml

```diff
@@ -11,6 +11,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Install Northfear SDL fork
+        run: |
+          chown -R 1000:1000 northfear-sdl2
+          cd northfear-sdl2
+          sudo --preserve-env=VITASDK -u \#1000 vita-makepkg
+          vdpm sdl2-2.30.2-1-arm.tar.xz
```

Comment:
What about something like:

```
PKGVER=$(grep pkgver= VITABUILD | cut -d '=' -f2)
PKGREL=$(grep pkgrel= VITABUILD | cut -d '=' -f2)
vdpm sdl2-$PKGVER-$PKGREL-arm.tar.xz
```

This way we don't need to remember to update the sdl2 version everywhere

Reply:
I decided to go with the easiest solution and use glob, since there is only one such file in the folder :)

Reply:
Amazing!